iaac/terraform/common/ack-sagemaker-controller/main.tf

```diff
@@ -1,7 +1,12 @@
+data "github_repository_file" "profile_controller_policy" {
+  repository = "awslabs/kubeflow-manifests"
+  branch     = "main"
```

Comment:
Please specify a git hash instead of a branch which could change.  This could break us in the future without notice.

Reply:
We can't use tags yet (https://github.com/integrations/terraform-provider-github/issues/1517) but we can use a branch. We can also specify our "ni" version in the repo if we want. Thoughts?

Reply:
locked to "ni" repo